### PR TITLE
Add sanity checks to required signatures.

### PR DESCRIPTION
**Pull request by lastperson**
Make sure m_required doesn't set to incorrect values.

### CODE REVIEW

File: Wallet.sol [hunk 1]
```diff
@@ -118,6 +120,7 @@ contract multiowned {
 	}
 
 	function changeRequirement(uint _newRequired) onlymanyowners(sha3(msg.data)) external {
+		if (_newRequired == 0) return;
```
Comment (gavofyork) [inline Wallet.sol]: `require(_newRequired > 0);`?

**Comment by gavofyork** [comment]
looks good other than the minor suggestion.

File: Wallet.sol (hunk 1 above)
Comment (lastperson) [inline Wallet.sol]: I thought so too, then decided to follow the convention to not burn the gas without the need, like on the next line:
`if (_newRequired > m_numOwners) return;`

File: Wallet.sol [hunk 2]
```diff
@@ -56,6 +56,8 @@ contract multiowned {
 	// as well as the selection of addresses capable of confirming them.
 	function multiowned(address[] _owners, uint _required) {
 		require(_owners.length > 0);
+		require(_owners.length >= _required);
+		require(_required > 0);
```
Comment (ngotchac) [inline Wallet.sol]: `require(_owners.length > 0);` is redundant then

**Comment by lastperson** [comment]
@ngotchac I agree!

File: Wallet.sol (hunk 1 above)
Comment (gavofyork) [inline Wallet.sol]: fair enough

**Pull request merged**